crates/common/consensus/src/fork_choice/helpers/constants.rs

```diff
@@ -11,8 +11,9 @@ pub const DOMAIN_BEACON_ATTESTER: B32 = fixed_bytes!("01000000");
 pub const DOMAIN_BEACON_PROPOSER: B32 = fixed_bytes!("00000000");
```

Comment:
```suggestion
pub const DOMAIN_BEACON_ATTESTER: B32 = fixed_bytes!("0x01000000");
pub const DOMAIN_BEACON_PROPOSER: B32 = fixed_bytes!("0x00000000");
pub const DOMAIN_BLS_TO_EXECUTION_CHANGE: B32 = fixed_bytes!("0x0A000000");
pub const DOMAIN_DEPOSIT: B32 = fixed_bytes!("0x03000000");
```
I just noticed that L10-L13 don't have hex prefix. I believe we want consistency here.

Reply:
on it